packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -592,8 +592,16 @@ contract OPContractsManagerUpgrader is OPContractsManagerBase {
     /// @notice Thrown when the SuperchainConfig contract does not match the unified config.
     error OPContractsManagerUpgrader_SuperchainConfigMismatch();
 
+    /// @notice The OPContractsManagerUpgrader contract.
+    OPContractsManagerUpgrader public immutable thisOPCMUpgrader;
+
+    /// @notice Mapping of SuperchainConfig addresses to if it has been upgraded or not.
+    mapping(address => bool) public isSuperchainConfigUpgraded;
```

Comment:
We should put the mapping into the outer `OPContractsManager` contract (after `isRC`), then we won't need to add `thisOPCMUpgrader`.